### ISSUE TITLE
Supporting `paragraph` in `role` atttribute

### ISSUE THREAD

**Issue by qmg-akara** [validator/validator#1432]
Hi guys, thanks for the great project!

I'm getting the following error for `<div role"paragraph">` but looks like `paragraph` is a [valid](https://github.com/w3c/aria/issues/474) value for `role`.

URL being validated or code to reproduce error:

```
<div role="paragraph">
Text...
</div>
```

```
Error: Bad value paragraph for attribute role on element div.
```
Am I missing anything?

**Comment by scottaohara** [comment]
yes, this role is new to aria 1.2 and should be valid.

also, i do truly hope this is not code that is going to find its way into production.

**Comment by qmg-akara** [comment]
Hey @scottaohara , thanks for the reply. 
I'm not sure what you mean by: `also, i do truly hope this is not code that is going to find its way into production.`
Do you mean the following code:
```html
<div role="paragraph">
Text...
</div>
```
If so, then I wonder if there is any rules for a **valid paragraph**.

_PS: the above code is just for testing the validator_

**Comment by scottaohara** [comment]
yes, i meant that code.  `parapgraph` was added to ARIA for the purposes of role parity with HTML. It's honestly not really a role that I would have ever expected to see someone use in the wild.

In other words your code is a perfect demonstration of over-engineering the `<p>` element, which is why I'd hope that someone would instead write `<p>`, rather than `<div role="paragraph">`

**Comment by qmg-akara** [comment]
Ah I see, that's a good point, thanks!

I replaced `div` with `p` and for testing I added `role="paragraph"`. But validator still complains about this code:
```
<p role="paragraph">
Text...
</p>
```
I can write tests without `role="paragraph"` but I wonder if the above usage is still not valid?

**Comment by scottaohara** [comment]
there'd be no reason to do `<p role=paragraph>`.  a `<p>` is already an implicit `role=paragraph`.  

Again, the validator should be updated to not flag an error for `<div role=paragraph>`, even though it's silly for anyone to do this.  

The validator _should_ expose a warning for `<p role=paragraph>` as an unnecessary redundancy

**Comment by scottaohara** [comment]
@atyposh that's fair if it makes sense and is supported for your use case.  my comment was only based on writing that sort of markup for the web - where if you did what you're suggesting, then screen readers behave inconsistently and can completely ignore parts of the paragraph.

**Comment by atyposh** [comment]
Hi @scottaohara, we are writing that sort of markup for the web. I wasn't aware that doing so would adversely affect screen readers. Our sites (hundreds of them using `<div role="paragraph">` extensively) are regularly audited. The only problem that is reported is that "paragraph" is not a valid role; which I'm now afraid may be masking other issues.

Thanks for this tip. We'll be on the lookout for assistive technology ignoring parts of paragraphs. But until we get past the validity issues, it'll be hard to test further.
